src/twig.logic.js

```diff
@@ -616,10 +615,7 @@ module.exports = function (Twig) {
                 }
 
                 // Render the template in case it puts anything in its context
-                template.render(innerContext);
-
-                // Extend the parent context with the extended context
-                Twig.lib.extend(context, innerContext);
```

Comment:
So I had this extend in for a reason, see https://github.com/twigjs/twig.js/pull/369.

What did you manage to do instead to make the tests pass?